### ISSUE TITLE
FAnim component running slowly

### ISSUE THREAD

**Issue by SolarLune** [GoranM/bdx#158]
Hey, there. 

So basically, it seems like the FAnim component runs slowly (like, at a snail's pace). Setting the frame directly seems to work correctly, and the animation plays back at the same speed in Blender as it does in BDX.

I've noticed that `increment` is set to 5/60. Is this intentional? Shouldn't it be 1 (advance a frame in the animation every frame in the game)?

**Comment by GoranM** [comment]
> I've noticed that increment is set to 5/60. Is this intentional? Shouldn't it be 1 (advance a frame in the animation every frame in the game)?

It should be the blender animation framerate (24 by default) over 60.

I didn't really finish the FAnim component, so I had a hardcoded value I used for testing, instead of actually exporting, and subsequently using the blender animation framerate.

A quick fix would be to change increment to 24/60, and add an fps method which could set it to whatever you prefer at runtime.

**Comment by SolarLune** [comment]
Are you sure about the integer rate being the default blender animation framerate? Seems like it would best be served either by passing the data from Blender's exporter out, or by just going with a value of 1 and letting the developer know in documentation that the animation needs to be adjusted.

Doesn't seem to make sense to play back an animation at 24 FPS in a 60 FPS game. Won't the animation be choppier in-engine?

Also, a frame() function to return the frame the FAnim component is on would be useful.

**Comment by GoranM** [comment]
> Doesn't seem to make sense to play back an animation at 24 FPS in a 60 FPS (by default) game. Won't the animation be choppier in-engine?

It shouldn't be - I mean, it's all done with interpolation, implemented via bezier curves.

Looks fine in my tests.

**Comment by SolarLune** [comment]
OK, PR submitted. 

I moved the increment fps call out of `action()` into the constructor so that the FPS wouldn't automatically reset every time you change the action without warning. I added an fps() function, as you suggested, and also a frame() getter to return the frame the animation is on. I also added an `isPlaying()` function to return if an FAnim is running (seems liked it'd be easier than using `if anim.state != null`).
